hi-ml-histopathology/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Using the histopathology models
+
+## Onboarding to Azure
+
+Prio 1, to be completed later
+
+## Creating datasets
+
+* [How to get PANDA (whole slides) and upload it to Azure](panda_dataset.md)
+* [How to get TCGA-Crck (tiles) and upload it to Azure](tcga-crck_dataset.md)
+
+## Training a model
+
+* Prio 0: [PANDA DeepMil on whole slides with ImageNet encoder](panda_model.md)
+  * How to invoke training for the existing model on PANDA data
+  * Where to find results and how to interpret them
```

Comment:
Here do we mean to explain how to use Azure ML dashboards, or our own pipeline's outputs? Or both?